windows: Properly escape command line arguments (#8627)
alacritty/alacritty: alacritty_terminal/CHANGELOG.md

## 0.25.1-dev

### Added

> - Added `escape_args` field to `tty::Options` for Windows shell argument escaping control

Reviewer: ```suggestion
- New `escape_args` field on `tty::Options` for Windows shell argument escaping control
```
